src/layer/vector/SVG.js

```diff
@@ -48,31 +47,30 @@ export const SVG = Renderer.extend({
 	},
 
 	_destroyContainer() {
-		this._container.remove();
-		DomEvent.off(this._container);
-		delete this._container;
+		Renderer.prototype._destroyContainer.call(this);
 		delete this._rootGroup;
 		delete this._svgSize;
 	},
 
+	_resizeContainer() {
+		const size = Renderer.prototype._resizeContainer.call(this);
+
+		// set size of svg-container if changed
+		if (!this._svgSize || !this._svgSize.equals(size)) {
+			this._svgSize = size;
+			this._container.setAttribute('width', size.x);
+			this._container.setAttribute('height', size.y);
+		}
+	},
+
 	_update() {
 		if (this._map._animatingZoom && this._bounds) { return; }
 
-		Renderer.prototype._update.call(this);
-
 		const b = this._bounds,
 		    size = b.getSize(),
 		    container = this._container;
 
-		// set size of svg-container if changed
-		if (!this._svgSize || !this._svgSize.equals(size)) {
-			this._svgSize = size;
-			container.setAttribute('width', size.x);
-			container.setAttribute('height', size.y);
-		}
-
 		// movement: update container viewBox so that we don't have to change coordinates of individual layers
-		DomUtil.setPosition(container, b.min);
```

Comment:
This call (and the `L.Canvas` equivalent) is redundant - the `_updateTransform` method from `L.Renderer` (now moved to `BlanketOverlay`) takes care of that.